provisioning/Samples/device/SymmetricKeySample/SymmetricKeySample.csproj

```diff
@@ -2,15 +2,16 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <LangVersion>8.0</LangVersion>
+    <RootNamespace>Microsoft.Azure.Devices.Provisioning.Client.Samples</RootNamespace>
   </PropertyGroup>
   <ItemGroup>
-    <Compile Include="..\Common\ProvisioningDeviceClientSample.cs" Link="ProvisioningDeviceClientSample.cs" />
```

Comment:
The (provision with dps + test with IoT hub) lived in this common code, with these individual samples showing how to create the different types of security providers. Do we want to deviate from that approach?

Reply:
Yes, I do. Having all the extra code about non-symmetric key for a symmetric key sample is distracting.